src/app/app.module.ts

```diff
@@ -50,20 +55,22 @@ import { BreadcrumbComponent } from "./breadcrumb/breadcrumb.component";
     SearchNavComponent,
     CscComponent,
     ProjectResourceComponent,
-    TitleLinkComponent,
     BreadcrumbComponent,
   ],
+  bootstrap: [AppComponent],
   imports: [
     LeafletModule,
     BrowserModule,
     FormsModule,
-    HttpClientModule,
     routing,
     NgSelectModule,
-    Ng2SmartTableModule,
     NgbDropdownModule,
     NgbModule,
     FontAwesomeModule,
+    MatTableModule,
+    MatSortModule,
+    MatCheckboxModule,
+    BrowserAnimationsModule,
```

Comment:
Remove if unused

Reply:
Those are all used for Angular Material for both the tables and checkboxes.